_visualisation/core/interests-arrows.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Some questions raised for me by this page:

(1) Do we say that an interest line **must** terminate with an arrow head, or is the arrow head optional? 

I would prefer optional, as hand drawing diagrams, and in some diagram tools, it's easier without. However, we can certainly say that arrows, if used, must only ever go in the one direction. 

(2) Do we need to say / show anything about where on the node the arrow can connect?

I.e. All our examples tend to show arrows connecting neatly to a single contact point on the edge, but what do we need rules to govern the following examples:

### OK (I think)

![image](https://user-images.githubusercontent.com/342624/84677890-4128ce00-af27-11ea-8d57-d2b92110b9d4.png)

### Not OK

![image](https://user-images.githubusercontent.com/342624/84677915-4ab23600-af27-11ea-8da3-a65f3e5a51e3.png)


### Is it OK? 

![image](https://user-images.githubusercontent.com/342624/84677944-5271da80-af27-11ea-93a3-d41b8a62739e.png)

The examples don't show any like this, so it's not 100% clear if this is good practice or not. 

Note: the visualisation tool I think currently works by attaching lines to a point at the centre of the node, so generates diagrams like the above.

Reply:
@timgdavies All of the above are unambiguous, but some are visually clearer than others. So should it be a _rule_ of BOVS that you have to use the nicer option, or do we leave it up to individual designers?

My intuition is:

* Arrowheads are mandatory. Yes it's a little more to draw, but not that hard, and it ensures that the direction of flow is completely clear, even if you're zoomed in, say, on a section of the diagram. The arrow also clarifies the meaning of an active voice annotation like "Owns".
* Where you connect the arrow isn't specified. The example diagrams keep things clean by connecting at one point, and people may choose to copy that, but for now that's just a stylistic choice.

Reply:
@Convincible - sure. Let's go with that.